src/test/java/net/library/controller/BookControllerTest.java

```diff
@@ -33,17 +39,27 @@ public BookControllerTest(BookService service) {
         this.service = service;
     }
 
+    @BeforeAll
+    static void setup() {
+        postgreSQLContainer = PostgresTestContainerConfig.getPostgreInstance();
```

Comment:
you should init db and upload data before each start up in the test

Reply:
+